src/commands/kill.c

```diff
@@ -23,6 +23,19 @@
 
 #include "command.h"
 
+static gboolean allProcesses = false;
+
+static GOptionEntry options_kill[] =
+{
+	{
+		"all", 'a' , G_OPTION_FLAG_NONE,
+		G_OPTION_ARG_NONE, &allProcesses,
```

Comment:
all_processes. CamelCase is usually only for types in the glib/GNOME world, variables remain with a _